assets/scss/external-link-icon.scss

```diff
@@ -0,0 +1,20 @@
+$external-icon: '\e900';
+
+@mixin external-icon {
+  content: $external-icon;
+  font-family: 'icomoon';
+  color: $textcolor-coloured;
+}
+
+.is-external-link {
+  // Add external link icon to end paragraph in Card Components
+  &.card-link {
+    p:last-child {
+      &:after {
+        @include external-icon;
+        margin-left: 4px;
+        vertical-align: middle;
+      }
+    }
+  }
+}
```

Comment:
This is duplication of code, as there is already the icons.scss. I am more inclined to say: stick to that CSS and naming convention. So for example, add a `<span class="icon-external-link"></span>`. And I might prefer to have the icon after the main title on the cards.

Reply:
To achieve this you would have to inject the span element with javascript, thats why I have taken the approach of using pseudo-classes here. The span is ideal for when we can manually put the icon next to a title (for example), but when it comes to dynamically generated content, using classes is probably the best way to go. 

For example:

Cards components - the icon will likely appear after the last paragraph
Footer - the icon will appear after the link text

Added classes to the SmartLink gives us a lot of flexibility on where we want the icon to appear.

Reply:
I see your point, you are right. I still think however that we should be able to reduce duplication of code and/or make use of icomoons naming conventions for classes. I played around a bit, there is a branch here: https://github.com/europeana/portal.js/tree/feat/EC-3835-external-links-reuse-css

What do you think of this approach?

Reply:
@mirjamverloop can you please show me what you have done please?